package.json

```diff
@@ -47,6 +47,18 @@
   "engines": {
     "node": ">=18.17.0"
   },
+  "release": {
+    "plugins": [
+      ["@semantic-release/commit-analyzer", {
+        "releaseRules": [
+          { "type": "docs", "release": "patch" }
+        ]
+      }],
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/npm",
+      "@semantic-release/github"
```

Comment:
These four are the default plugins for semantic-release, in the same order. We need to include them all here, even though we are only configuring commit-analyzer, as the order in which they are defined is critical.